### ISSUE TITLE
BUG - sev-3 - All - Onboarding bullet points have insufficient contrast in light mode (new in 2.41)

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#10135]
## What happened?

During RC testing for 2.41, I am seeing that in light mode only, bullet point icons have insufficient contrast with the dark blue background:
<img width="368" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeWFqQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--181d308cf8638b704bbed73e2b228c570551861d/image.png" alt="image.png" />

Confirmed that dark mode has white bullet points (aka sufficient contrast) and that this is new in this release. I'm guessing it was caused by the icon conversion [(ticket 9779).](https://github.com/department-of-veterans-affairs/va-mobile-app/issues/9779)

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by theodur** [comment]
This issue was fixed in #9199. After @TKDickson brought up @Sparowhawk's PR, I was going to revert the commit I pushed to my PR that fixes this, but I forgot to do that. So the fix for this is `develop` now

**Comment by TKDickson** [comment]
@rbontrager FYI since this is a global ticket
